src/gitpodify.ts

```diff
@@ -70,13 +70,13 @@ const injectedByUserClick = (): boolean => {
 }
 
 const updateOnDOMChanges = (injector: Injector) => {
-    observer = new MutationObserver(function (mutations) {
-        if (!injector.checkIsInjected() && !isInstalling) {
-            isInstalling = true;
-            injector.update()
-                .then(() => isInstalling = false);
-        }
-    });
+    observer = new MutationObserver(() => {
+            if (!injector.checkIsInjected() && !isInstalling) {
+                isInstalling = true;
+                injector.update()
+                    .then(() => isInstalling = false);
+            }
+        });
```

Comment:
@filiptronicek I think there are more spaces for indentation 😄 

![image](https://user-images.githubusercontent.com/69431456/140647131-fef5283c-ad4a-44be-be85-e95ecd3bf4d6.png)